amf-aml/shared/src/main/scala/amf/plugins/document/vocabularies/model/domain/SemanticExtension.scala

```diff
@@ -10,11 +10,11 @@ import org.yaml.model.YMap
 case class SemanticExtension(fields: Fields, annotations: Annotations) extends DomainElement {
   override def meta: SemanticExtensionModel.type = SemanticExtensionModel
 
-  def extensionName(): StrField                       = fields.field(ExtensionName)
-  def extensionMappingDefinition(): AnnotationMapping = fields.field(ExtensionMappingDefinition)
+  def extensionName(): StrField              = fields.field(ExtensionName)
+  def extensionMappingDefinition(): StrField = fields.field(ExtensionMappingDefinition)
 
   def withExtensionName(name: String): SemanticExtension = set(ExtensionName, name)
-  def withExtensionMappingDefinition(annotationMapping: AnnotationMapping): SemanticExtension =
+  def withExtensionMappingDefinition(annotationMapping: String): SemanticExtension =
```

Comment:
Guessing you moved this to strings to checkNodeMappableReferences

Reply:
Yes, I made AnnotationMapping extend NodeMappable to do this.